src/p2p/index.ts

```diff
@@ -89,18 +95,45 @@ export class P2PNetwork extends EventEmitter implements Service {
 
       });
 
+      // 2-way handshake
+      const protocol: string = "/eth/serenity/beacon/rpc/1";
+      const helloMsg: Hello = {
+       
```

Comment:
you should be able to fill this object

Reply:
I know. I'm just leaving that there as a placeholder while I get the other pieces in place.